.github/MAINTAINERS_GUIDE.md

```diff
@@ -766,7 +766,7 @@ Steps to merge a pull request:
 When in doubt, find the other maintainers and ask.
 
 [circleci]: ../.circleci/config.yml
-[commands]: https://api.slack.com/automation/cli/commands
+[commands]: https://tools.slack.dev/slack-cli/reference/commands/slack
```

Comment:
note: We might be able shorten this URL in the future to:
https://tools.slack.dev/slack-cli/reference/commands